ygen/gogen.go

```diff
@@ -300,6 +300,20 @@ type generatedLeafGetter struct {
 	Receiver string
 }
 
+// generatedDefaultMethod is used to represent parameters required to generate
+// a PopulateDefaults method for a GoStruct that recursively populates default
+// values within the subtree.
+type generatedDefaultMethod struct {
+	// Receiver is the name of the receiver for the default method.
+	Receiver string
+	// ChildContainerNames are the names of the container fields of the GoStruct.
+	ChildContainerNames []string
+	// ChildContainerNames are the names of the list fields of the GoStruct.
+	ChildListNames []string
+	// Leaf represent the leaf fields of the GoStruct.
```

Comment:
Comment should use the same name as the field. Suggest `Leaves` or `Leaf`.

Reply:
Done.